apps/wallet-mobile/src/features/RampOnOff/common/ShowBuyBanner/BuyBannerBig.tsx

```diff
@@ -34,7 +32,7 @@ export const BuyBannerBig = () => {
         style={styles.gradient}
         start={{x: 1, y: 1}}
         end={{x: 1, y: 1}}
-        colors={theme.color.gradients['green']}
+        colors={styles.gradientColor as (string | number)[]}
```

Comment:
@SorinC6 we need to type the colors in the style with the StyleSheet color type, so we can avoid this casting.